helix-term/src/ui/prompt.rs

```diff
@@ -447,7 +447,7 @@ impl Prompt {
             text.render(inner, surface, cx);
         }
 
-        let line = area.height - 1;
+        let line = area.height - 2;
```

Comment:
This and the following change on line 642 break the file picker (and probably other prompts as well), since they can underflow.
If you substitute this and the change on line 642 with `saturating_sub(2)` this seems to fix it.